app/components/page_list_component/_index.scss

```diff
@@ -9,5 +9,18 @@
 }
 
 .app-page-list__key {
-  width: 15%;
+  @include govuk-media-query($from: desktop) {
+    width: 20%;
+  }
+}
+
+.govuk-form-group--error .app-page-list__key {
+  @include govuk-media-query($from: tablet) {
+    padding-left: govuk-spacing(3);
+  }
+}
+
+.app-page_list__route-text--error:link,
+.app-page_list__route-text--error:visited {
+  color: $govuk-error-colour;
```

Comment:
Doesn't this cause an A11y issue? Being able to identify if a user has visited a page before?